src/config/momo.config.ts

```diff
@@ -7,20 +7,22 @@ const secreteKey = 'oimuCa81icwpHAgnw44iHKiiK8sSX5dM';
 const momoEndpoint = 'https://test-payment.momo.vn/gw_payment/transactionProcessor';
 const orderInfo = 'pay with Momo';
 const requestType = 'captureMoMoWallet';
-const notifyUrl = 'https://a0fd3e739794.ngrok.io/v1/payment/momo';
-const returnUrl = 'http://localhost:3000/app/dashboard';
+const notifyUrl = 'http://localhost:8000/v1/payment/momo';
+const returnUrl = 'http://localhost:3000/app/appointment';
 
 const momoRequest = async (
-  appointmentId: string,
+  scheduleId: string,
   appointmentName: string,
   patientId: string,
   patientName: string,
+  startOfAppointment: string,
+  endOfAppointment: string,
   doctorId: string,
   doctorName: string,
   amount: string,
 ) => {
   const orderId = uuid();
-  const extraData = [appointmentId, appointmentName, patientId, patientName, doctorId, doctorName].join(',');
+  const extraData = [scheduleId, appointmentName, patientId, patientName, startOfAppointment, endOfAppointment, doctorId, doctorName].join(',');
```

Comment:
vì sao lại sửa appointmentId thành schedule vậy Bảo, có hiểu rõ mục đích của extraData đó k

Reply:
cái ni để lấy cái schedule nớ qua return page update lại status cho cái schedule nớ việt, với appointmentId thì tự động gen sau khi thanh toán thành công chứ chưa có sẵn để truyền vô đây

Reply:
thế lúc tạo thanh toán, thì sẽ lấy cái gì để liên kết giữa Appointment và Payment ?